Makefile

```
@@ -152,6 +152,15 @@ else
 endif
 endif
 
+i18n-extract-server:
+	goi18n extract -format json -outdir assets/i18n/ server/
+	touch assets/i18n/translate.es.json
+	goi18n merge -format json -outdir assets/i18n/ assets/i18n/active.*.json assets/i18n/translate.es.json
```

Can this command be generic enough to support other Landeslage then `es`?
```suggestion
	goi18n merge -format json -outdir assets/i18n/ assets/i18n/active.*.json
```

---

I'll try to do this, I'm not verify familiarized with the `goi18n` tool yet

---

Done